README.md

```diff
@@ -64,6 +64,11 @@ You can access Nginx and Symfony application logs in the following directories o
 
 You can also use Kibana to visualize Nginx & Symfony logs by visiting `http://symfony.localhost:81`.
 
+# Use xdebug!
+
+To use xdebug change the line `"docker.host:127.0.0.1"` in docker-compose.yml and replace 127.0.0.1 with your machines ip addres.
```

Comment:
Great but can you fix typos here please:
`your machines ip addres` -> `your machine ip address`